images/dockerfiles/telegraf-datadog-forwarder-testnet/telegraf.conf.tmpl

```diff
@@ -239,7 +239,18 @@
   ## Maximum line size allowed to be sent in bytes.
   ## 0 means to use the default of 65536 bytes (64 kibibytes)
   max_line_size = 0
-  
+
+[[inputs.exec]]
+  # This input runs the scripts below which generates connectivity info every 10 minutes
+  interval = "1s"
```

Comment:
why do we need to run this every second? can we do this once every 10s, which is telegraf default report interval anyways?

Reply:
the agent is sampling every 5s, I changed the input interval to 5s so it will match the agent